### ISSUE TITLE
[bug] Switching orientation breaks the AR

### ISSUE THREAD

**Issue by marcusx2** [hiukim/mind-ar-js#238]
If I switch the orientation on the phone from portrait to landscape and vice versa, the augmented content disappears permanently.

**Comment by marcusx2** [comment]
Apparently this bug was already reported [here](https://github.com/hiukim/mind-ar-js/issues/212). Will it be fixed?

**Comment by marcusx2** [comment]
@hiukim Any thoughts on this? Will it be fixed, or are there currently no plans to fix it?

**Comment by hiukim** [comment]
Yes, currently it will break if device changes orientation. I agree it needs to be handled. I will look into it when I have some free time.

**Comment by marcusx2** [comment]
Ah ok. For now I'll need to use [zappar](https://www.zappar.com/). ARJS' image tracking is just not good enough, and switching orientation is really important for production. I can't just tell the client not to switch orientations T_T. I hope this gets fixed soon

**Comment by marcusx2** [comment]
Any news? ;_;

**Comment by marcusx2** [comment]
@hiukim Will this be fixed soon? ;_;

**Comment by hiukim** [comment]
@marcusx2 sorry, unlikely. At least not these two weeks. Busy with work lately.

**Comment by marcusx2** [comment]
Is there any workaround while this isn't fixed?

**Comment by arslc** [comment]
@marcusx2 try this:

```javascript
window.addEventListener('orientationchange', () => {
 //stop mindar
 mindar.stop()

 // ur function to hide 3D object (optional)

 //restart mindar after 1 second delay
 setTimeout(()=>{
  mindar.start()
 }, 1000)

}, false)
```
Not the most elegant way, but as a temporary solution it may help.

*P.S. I know, this is deprecated event, but it still work on iOS 16.
So we can use that until @hiukim not fix this bug*

**Comment by marcusx2** [comment]
@arslc Thanks this is a good enough workaround, it works just fine!

**Comment by arslc** [comment]
@marcusx2  Glad it helped :)
@hiukim Probably should be closed issue?

**Comment by marcusx2** [comment]
@arslc I don't think it should be closed. I think we should be able to rotate the phone without having to stop and start the AR engine again. This is still just a workaround in my opinion.

**Comment by hiukim** [comment]
fixed in v1.2.2
